monitoring/overview-of-monitoring.adoc

```diff
@@ -0,0 +1,37 @@
+[id="overview-of-monitoring"]
+= Overview of monitoring
+include::modules/common-attributes.adoc[]
+:context: overview-of-monitoring
+
+toc::[]
+
+// About cluster monitoring (the title is only shown in virt/logging_events_monitoring/virt-openshift-cluster-monitoring.adoc)
+include::modules/monitoring-about-cluster-monitoring.adoc[leveloffset=+1]
+
+// Understanding the monitoring stack
+include::modules/understanding-the-monitoring-stack.adoc[leveloffset=+1]
+include::modules/monitoring-default-monitoring-components.adoc[leveloffset=+2]
+include::modules/monitoring-default-monitoring-targets.adoc[leveloffset=+2]
+include::modules/monitoring-components-for-monitoring-user-defined-projects.adoc[leveloffset=+2]
+include::modules/monitoring-targets-for-user-defined-projects.adoc[leveloffset=+2]
+
+[id="how-to-use-the-monitoring-stack"]
+= How to use the monitoring stack?
```

Comment:
My feeling is that this new section duplicates a lot of the content that is included in the "About OpenShift Container Platform monitoring" section at the start of the assembly.

If the intention is to include links to the various resources included within the Monitoring book, you could move the existing "About OpenShift Container Platform monitoring" section from `modules/monitoring-about-cluster-monitoring.adoc` to this `monitoring/overview-of-monitoring.adoc` assembly and add the cross references to the existing content.

My view is that it makes sense also to keep that section at the start of the assembly as it provides a useful introduction for the reader.

Reply:
@pneedle-rh Are you saying remove the `include::modules/monitoring-about-cluster-monitoring.adoc[leveloffset=+1]` at the start of the assembly and add it as an xref in the new section `How to use the monitoring stack?`. Yes the objective is to give an overall view/relevant links to the entire book.

Reply:
I was suggesting that you could move the entire contents of `modules/monitoring-about-cluster-monitoring.adoc` from that module file into the `modules/about-monitoring-stack-components.adoc` assembly itself. With the contents being in the assembly instead of the module, you could then add cross-references within the existing content, to the different pages in the Monitoring book. This would remove the need for an additional section such as the "How to use the monitoring stack?" section, which duplicates a lot of the contents that are in "About OpenShift Container Platform monitoring".

In other words, instead of adding the "How to use the monitoring stack?" section into the assembly, you could have the "About OpenShift Container Platform monitoring" section in the assembly (not in a module) and include cross-references to other pages in that existing content.

Reply:
It's just an idea for your consideration, to reduce duplication in the page.

Reply:
@ctauchen what are your thoughts on this? Thanks!

Reply:
Looking at the draft as it stands now, I noticed the duplication between the sections. This wasn't something I had considered previously and I think it is worth trying to minimise that. My apologies if my further suggestions have created additional work for you, that was not my intention.

Reply:
On further thought, I like your suggestion of adding the content from the module` modules/monitoring-about-cluster-monitoring.adoc` and then adding what's relevant from the new section `[id="how-to-use-the-monitoring-stack"]`.

Reply:
> @ctauchen what are your thoughts on this? Thanks!

I think that's a good solution. Incorporate the new content cleanly into the existing structure. It's slightly messy to put it all in the assembly. With tools limitations, that's probably the best option.

Nearly there!